src/JBMultiTerminal.sol

```diff
@@ -678,6 +682,17 @@ contract JBMultiTerminal is JBPermissioned, ERC2771Context, IJBMultiTerminal {
             metadata: metadata
         });
 
+        // Get a reference to the beneficiary's balance after the payment.
+        uint256 beneficiaryBalanceAfter = tokens.totalBalanceOf({holder: beneficiary, projectId: projectId});
+
+        // Set the beneficiary token count.
+        if (beneficiaryBalanceAfter <= beneficiaryBalanceBefore) {
+            beneficiaryTokenCount = 0;
+        }
+        else {
+            beneficiaryTokenCount = beneficiaryBalanceAfter - beneficiaryBalanceBefore;
+        }
+
```

Comment:
We can simplify this code and the statement below this into something like
```
if (minReturnedTokens != 0) {
       // Get a reference to the beneficiary's balance after the payment.
        uint256 beneficiaryBalanceAfter = tokens.totalBalanceOf({holder: beneficiary, projectId: projectId});
        
        // The token count for the beneficiary must be greater than or equal to the specified minimum.
        if ( beneficiaryBalanceBefore >= beneficiaryBalanceAfter || beneficiaryBalanceAfter - beneficiaryBalanceBefore < minReturnedTokens) {
            revert JBMultiTerminal_UnderMinReturnedTokens(beneficiaryTokenCount, minReturnedTokens);
      } 
}

Reply:
we still gotta return `beneficiaryTokenCount` tho